.github/workflows/deploy-DEV.yml

```diff
@@ -259,7 +234,7 @@ jobs:
         run: cd mega-linter-runner && sudo yarn install --frozen-lockfile && sudo npm link
       - name: Run mega-linter-runner tests
         if: ${{ steps.docker_build.outcome }} == 'success' && !contains(github.event.head_commit.message, 'quick build')
-        run: cd mega-linter-runner && MEGALINTER_RELEASE=${{ steps.image_tag.outputs.tag }} MEGALINTER_NO_DOCKER_PULL=true npm run test
+        run: cd mega-linter-runner && MEGALINTER_RELEASE=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }} MEGALINTER_IMAGE=${{ fromJson(steps.meta.outputs.json).tags[0]}} MEGALINTER_NO_DOCKER_PULL=true npm run test
```

Comment:
The MEGALINTER_IMAGE is set here, and only used for the test scripts. Internal only. It’s the way I can add an argument to the options

Reply:
ok :)